tox.ini

```diff
@@ -6,7 +6,8 @@ whitelist_externals = env
 install_command = pip install -U {opts} {packages}
 setenv= TOX_ENV_NAME={envname}
 passenv = TOX_*
-deps = -r{toxinidir}/requirements.txt
+deps = {py,py26,py27,py33,py34,py35,pypy}: pip
```

Comment:
Based on this: https://testrun.org/tox/latest/config.html#generating-environments-conditional-settings

Would this work, without changing circle.yml ?

```
deps = py32: pip<8.0 virtualenv<14.0
       -r{toxinidir}/requirements.txt
       -r{toxinidir}/test-requirements.txt
```

I guess it probably wouldn't, since the virtualenv will already have been created. 

I'm just trying to think of a way to change the environment _only_ for the py32 tests, then it will be easier to revert the changes once the pip/virtualenv issues are fixed upstream.